diff --git a/node_modules/react-navigation-drawer/LICENSE b/node_modules/react-navigation-drawer/LICENSE
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/README.md b/node_modules/react-navigation-drawer/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/index.js b/node_modules/react-navigation-drawer/lib/commonjs/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/index.js.map b/node_modules/react-navigation-drawer/lib/commonjs/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/navigators/createDrawerNavigator.js b/node_modules/react-navigation-drawer/lib/commonjs/navigators/createDrawerNavigator.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/navigators/createDrawerNavigator.js.map b/node_modules/react-navigation-drawer/lib/commonjs/navigators/createDrawerNavigator.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerActions.js b/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerActions.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerActions.js.map b/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerActions.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerRouter.js b/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerRouter.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerRouter.js.map b/node_modules/react-navigation-drawer/lib/commonjs/routers/DrawerRouter.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/types.js b/node_modules/react-navigation-drawer/lib/commonjs/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/types.js.map b/node_modules/react-navigation-drawer/lib/commonjs/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerGestureContext.js b/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerGestureContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerGestureContext.js.map b/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerGestureContext.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerProgressContext.js b/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerProgressContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerProgressContext.js.map b/node_modules/react-navigation-drawer/lib/commonjs/utils/DrawerProgressContext.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/Drawer.js b/node_modules/react-navigation-drawer/lib/commonjs/views/Drawer.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/Drawer.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/Drawer.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerNavigatorItems.js b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerNavigatorItems.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerNavigatorItems.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerNavigatorItems.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerSidebar.js b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerSidebar.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerSidebar.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerSidebar.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerView.js b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerView.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerView.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/DrawerView.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/ResourceSavingScene.js b/node_modules/react-navigation-drawer/lib/commonjs/views/ResourceSavingScene.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/ResourceSavingScene.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/ResourceSavingScene.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/TouchableItem.js b/node_modules/react-navigation-drawer/lib/commonjs/views/TouchableItem.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/commonjs/views/TouchableItem.js.map b/node_modules/react-navigation-drawer/lib/commonjs/views/TouchableItem.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/index.js b/node_modules/react-navigation-drawer/lib/module/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/index.js.map b/node_modules/react-navigation-drawer/lib/module/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/navigators/createDrawerNavigator.js b/node_modules/react-navigation-drawer/lib/module/navigators/createDrawerNavigator.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/navigators/createDrawerNavigator.js.map b/node_modules/react-navigation-drawer/lib/module/navigators/createDrawerNavigator.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/routers/DrawerActions.js b/node_modules/react-navigation-drawer/lib/module/routers/DrawerActions.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/routers/DrawerActions.js.map b/node_modules/react-navigation-drawer/lib/module/routers/DrawerActions.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/routers/DrawerRouter.js b/node_modules/react-navigation-drawer/lib/module/routers/DrawerRouter.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/routers/DrawerRouter.js.map b/node_modules/react-navigation-drawer/lib/module/routers/DrawerRouter.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/types.js b/node_modules/react-navigation-drawer/lib/module/types.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/types.js.map b/node_modules/react-navigation-drawer/lib/module/types.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/utils/DrawerGestureContext.js b/node_modules/react-navigation-drawer/lib/module/utils/DrawerGestureContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/utils/DrawerGestureContext.js.map b/node_modules/react-navigation-drawer/lib/module/utils/DrawerGestureContext.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/utils/DrawerProgressContext.js b/node_modules/react-navigation-drawer/lib/module/utils/DrawerProgressContext.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/utils/DrawerProgressContext.js.map b/node_modules/react-navigation-drawer/lib/module/utils/DrawerProgressContext.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/Drawer.js b/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
old mode 100644
new mode 100755
index 3f81450..3b6b8ad
--- a/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
+++ b/node_modules/react-navigation-drawer/lib/module/views/Drawer.js
@@ -14,7 +14,7 @@ const {
   clockRunning,
   startClock,
   stopClock,
-  interpolate,
+  interpolateNode,
   spring,
   abs,
   add,
@@ -331,7 +331,7 @@ export default class Drawer extends React.PureComponent {
       onHandlerStateChange: this.handleTapStateChange
     }, /*#__PURE__*/React.createElement(Animated.View, {
       style: [styles.overlay, {
-        opacity: interpolate(this.progress, {
+        opacity: interpolateNode(this.progress, {
           inputRange: [PROGRESS_EPSILON, 1],
           outputRange: [0, 1]
         }),
diff --git a/node_modules/react-navigation-drawer/lib/module/views/Drawer.js.map b/node_modules/react-navigation-drawer/lib/module/views/Drawer.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerNavigatorItems.js b/node_modules/react-navigation-drawer/lib/module/views/DrawerNavigatorItems.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerNavigatorItems.js.map b/node_modules/react-navigation-drawer/lib/module/views/DrawerNavigatorItems.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerSidebar.js b/node_modules/react-navigation-drawer/lib/module/views/DrawerSidebar.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerSidebar.js.map b/node_modules/react-navigation-drawer/lib/module/views/DrawerSidebar.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerView.js b/node_modules/react-navigation-drawer/lib/module/views/DrawerView.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/DrawerView.js.map b/node_modules/react-navigation-drawer/lib/module/views/DrawerView.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/ResourceSavingScene.js b/node_modules/react-navigation-drawer/lib/module/views/ResourceSavingScene.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/ResourceSavingScene.js.map b/node_modules/react-navigation-drawer/lib/module/views/ResourceSavingScene.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/TouchableItem.js b/node_modules/react-navigation-drawer/lib/module/views/TouchableItem.js
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/module/views/TouchableItem.js.map b/node_modules/react-navigation-drawer/lib/module/views/TouchableItem.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/index.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/navigators/__tests__/createDrawerNavigator.test.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/navigators/__tests__/createDrawerNavigator.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/navigators/createDrawerNavigator.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/navigators/createDrawerNavigator.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/routers/DrawerActions.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/routers/DrawerActions.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/routers/DrawerRouter.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/routers/DrawerRouter.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/routers/__tests__/DrawerRouter.test.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/routers/__tests__/DrawerRouter.test.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/types.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/types.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/utils/DrawerGestureContext.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/utils/DrawerGestureContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/utils/DrawerProgressContext.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/utils/DrawerProgressContext.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/Drawer.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/Drawer.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerNavigatorItems.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerNavigatorItems.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerSidebar.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerSidebar.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerView.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/DrawerView.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/ResourceSavingScene.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/ResourceSavingScene.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/lib/typescript/src/views/TouchableItem.d.ts b/node_modules/react-navigation-drawer/lib/typescript/src/views/TouchableItem.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/index.tsx b/node_modules/react-navigation-drawer/src/index.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/navigators/__tests__/createDrawerNavigator.test.tsx b/node_modules/react-navigation-drawer/src/navigators/__tests__/createDrawerNavigator.test.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/navigators/createDrawerNavigator.tsx b/node_modules/react-navigation-drawer/src/navigators/createDrawerNavigator.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/routers/DrawerActions.tsx b/node_modules/react-navigation-drawer/src/routers/DrawerActions.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/routers/DrawerRouter.tsx b/node_modules/react-navigation-drawer/src/routers/DrawerRouter.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/routers/__tests__/DrawerRouter.test.tsx b/node_modules/react-navigation-drawer/src/routers/__tests__/DrawerRouter.test.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/types.tsx b/node_modules/react-navigation-drawer/src/types.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/utils/DrawerGestureContext.tsx b/node_modules/react-navigation-drawer/src/utils/DrawerGestureContext.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/utils/DrawerProgressContext.tsx b/node_modules/react-navigation-drawer/src/utils/DrawerProgressContext.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/Drawer.tsx b/node_modules/react-navigation-drawer/src/views/Drawer.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/DrawerNavigatorItems.tsx b/node_modules/react-navigation-drawer/src/views/DrawerNavigatorItems.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/DrawerSidebar.tsx b/node_modules/react-navigation-drawer/src/views/DrawerSidebar.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/DrawerView.tsx b/node_modules/react-navigation-drawer/src/views/DrawerView.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/ResourceSavingScene.tsx b/node_modules/react-navigation-drawer/src/views/ResourceSavingScene.tsx
old mode 100644
new mode 100755
diff --git a/node_modules/react-navigation-drawer/src/views/TouchableItem.tsx b/node_modules/react-navigation-drawer/src/views/TouchableItem.tsx
old mode 100644
new mode 100755
